src/widget/textarea/textarea.js

```diff
@@ -23,7 +25,16 @@ class TextareaWidget extends Widget {
                 this._resize( el );
             }
         } );
-        textareas.forEach( this._resize.bind( this ) );
+        // workaround for resize issue on first time load textarea with existing value on enketo-express
+        const mutationsTracker = new MutationsTracker(),
+        form = document.querySelector( 'form' ),
+        cls = 'first-load-textarea',
+        classChange = mutationsTracker.waitForClassChange( form, cls );
+        form.classList.add( cls );
+        classChange.then( () => {
+            textareas.forEach( this._resize.bind( this ) );
+            form.classList.remove( cls );
+        } );
```

Comment:
Would not adding the class, and instead using something like this work as well?:

```js
const mutationsTracker = new MutationsTracker(); 
mutationsTracker.waitForQuietness()
    .then( () => { .... }
```

If it does it may be worth checking if passing `this.element` to the constructor would work as well. This would help us in understanding the issue better (and I think using that would be better).

Reply:
yes, you're right it works. I just update the code also passing the `this.element` to the constructor. So this `.waitForQuietness` method will be fired after no more change happen to the el set on the contructor based on the list of mutation set on the MutationObserver, right?

CMIIW just want to share, I think there is a loophole in the logic in the `checkInterval` on MutationsTracker contructor
``const checkInterval = setInterval( () => {
            if ( previousMutations === mutations ){
                this.quiet = true;
                mutationObserver.disconnect();
                clearInterval( checkInterval );
            } else {
                this.quiet = false;
                previousMutations = mutations;
            }
        }, 100 );
``
from what I debug when try to learn it, it looks like the else condition never fired since the increment set on creation MutationObserver is not working(duplicate variable name).